src/language-js/printer-estree.js

```diff
@@ -2485,6 +2485,26 @@ function printPathNoParens(path, options, print, args) {
         path.call(print, "id"),
         path.call(print, "typeParameters")
       ]);
+
+    case "InterfaceType":
+    case "InterfaceTypeAnnotation": {
```

Comment:
Is there any way to combine this with the TS one?

Reply:
I was hesitant initially, since they can't have an `id` or `typeParameters`, but I went ahead and merged it with the logic for printing flow's `DeclareInterface` and `InterfaceDeclaration`.